woocommerce/woocommerce-android
# 7845 fix orientation change menu hiding

(contributed by @bunyodrafikov 🙏)

Fixed menu hiding when Lifecycle state changed

Closes: #7845 

### Description
In 27 Fragments in case you change orientation, the menu was not surviving and is broken. The issue was because of using Lifecycle.State.RESUMED which was deleted from all references

### Testing instructions
You can open 27 fragments that were changed to see how the new menu works after orientation changing

### Images/gif
For more images how it was and how it is working now, you can see this comment: [#7798 (comment)](https://github.com/woocommerce/woocommerce-android/pull/7798#discussion_r1021309336)
